src/lib/indexer/eventProcessor.ts

```diff
@@ -112,6 +112,26 @@ const DefaultEventProcessors: EventProcessors = {
       certificates: new Map([[local_id, update]]),
     }
   },
+  revoke_cert: ({ version, inputs, outputs }) => {
+    if (version !== 1) throw new Error(`Incompatible version ${version} for issue_cert process`)
+
+    const { local_id } = inputs[0]
+    const { id: latest_token_id, ...cert } = outputs[0]
+
+    const attachment: AttachmentRecord = attachmentPayload(cert.metadata, 'reason')
```

Comment:
The behaviour here looks incorrect. If it's our transaction we need to do something different. See https://github.com/digicatapult/dscp-matchmaker-api/blob/1ef781b85e1273de19f5347725e9ac5dc1191990/src/lib/indexer/eventProcessor.ts#L90 for an example